app/models/word.rb

```diff
@@ -11,4 +11,13 @@ class Word < ApplicationRecord
     conjunction: 6,
     interjection: 7
   }
+
+  # Validations
+  validates :content, presence: true
+  validates :meaning, presence: true
+  validates :word_type, presence: true, inclusion: {in: word_types.keys}
+
+  # Scopes
+  scope :by_type, ->(word_type){where(word_type:)}
```

Comment:
```suggestion
  scope :by_type, ->(word_type) {where(word_type:)}
```
Thêm 1 space trước block {}

Reply:
done